applications/luci-app-acme/htdocs/luci-static/resources/view/acme/acme.js

```diff
@@ -665,6 +657,45 @@ function _collectDdnsDomains() {
 	return ddnsDomains;
 }
 
+function _importDdns(ddnsDomains) {
+	alert(_('After import check the added domain certificate configurations.'));
+	let certSections = uci.sections('acme', 'cert');
+	let certSectionNames = new Map();
+	let certSectionDomains = new Map();
+	for (let s of certSections) {
+		certSectionNames.set(s['.name'], null);
+		if (s.domains) {
+			for (let d of s.domains) {
+				certSectionDomains.set(d, s['.name']);
+			}
+		}
+	}
+	console.log(certSections);
+	console.log(certSectionDomains);
+	for (let ddnsDomain of ddnsDomains) {
+		let sectionId = ddnsDomain.sectionId;
+		// ensure unique sectionId
+		if (certSectionNames.has(sectionId)) {
+			sectionId += '_' + new Date().getTime();
+		}
+		if (ddnsDomain.domains) {
+			for (let d of ddnsDomain.domains) {
+				let dupDomainSection = certSectionDomains.get(d);
+				if (dupDomainSection) {
+					alert(_('The domain %s in DDNS %s already was configured in %s. Please check it after the import.').format(d, sectionId, dupDomainSection));
```

Comment:
If it's already configured we should just skip it instead of bugging the user...

Reply:
Please remove this `alert()`